alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/ScheduledTask.java

```diff
@@ -46,36 +46,41 @@
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     private final TaskScheduler taskScheduler;
-    private final String taskName;
     private ScheduledFuture<?> future;
 
-    public ScheduledTask(final TaskScheduler taskScheduler, final String taskName) {
+    public static String computeTaskName(Class<? extends ScheduledTask> clazz) {
```

Comment:
Is this used by any other class should this be private

Reply:
It is used in the static method computeProviderTaskName() which is currently only called by the SchedulingGlobalApiAction.